faq.md

```diff
@@ -11,7 +11,7 @@ The following programming languages are currently supported:
 * C/C++
 * Go
 * Rust
-* Java HotSpot (JDK 7-16, AOT and compressed pointers are not supported)
+* Java HotSpot (JDK 7-16, AOT is not supported)
```

Comment:
@fabled my understanding was that we also support Java 17?

Reply:
Seems no commits after JDK16 support. But it is possible that JDK17 just works without any particular extra support. The hotspot code is fairly flexible.

Reply:
I actually thought about changing this as well but I found no explicit statement about JDK 17 in the code thus I left it unchanged.